main/telemetry.js

```diff
@@ -22,7 +22,9 @@ const writeClient = client.getWriteApi(
 
 setInterval(() => {
   writeClient.flush().catch(err => {
-    Sentry.captureException(err)
+    if (!/HttpError|getAddrInfo|RequestTimedOutError/i.test(String(err))) {
+      Sentry.captureException(err)
+    }
```

Comment:
Have you considered using a reverse logic & early return instead? It will make your change smaller.

```suggestion
    if (/HttpError|getAddrInfo|RequestTimedOutError/i.test(String(err))) return;
    Sentry.captureException(err)
```

It would be also nice to add a code comment to explain why are we ignoring these errors.